Telemetry/CorePing.swift

```diff
@@ -57,6 +57,8 @@ class CorePing: TelemetryPing {
 
         let locale = NSBundle.mainBundle().preferredLocalizations.first!.stringByReplacingOccurrencesOfString("_", withString: "-")
 
+        let defaultEngineId = profile.searchEngines.defaultEngine.engineId ?? "null"
```

Comment:
`"null"` -> `nil`. The former will be the string `"null"` in JSON, whereas the latter will be `null` (which is what we want).